### ISSUE TITLE
Hosting RChain on AMAfeed.com

### ISSUE THREAD

**Issue by philipandri002** [rchain/bounties#781]
### Benefit to RChain

*What is AMAfeed?

AMAfeed is a platform for knowledge exchange in an Ask Me Anything format. We make it easy for you to present yourself, your story, your expertise, project or a topic you are interested in and engage the audience in participating in a discussion about it.

Create new connections
Engage your existing following
Promote your service, expertise, project

It boast to have over 50 channels dedicated to different topics - from Artists and Authors, Technology, Crypto/Blockchain to Health,pet and many more.

How does it work?

The Interview. Reinvented.

An Ask Me Anything (AMA) Event is an authentic way to raise awareness to the things and causes that matter to you most.

As an AMA Host it is up to you how you utilize the platform. For example, you can:

Share your story to inspire and engage others
Whatever your story is – there is always someone who can benefit from it and learn from your ups and downs. And you – you get a great feedback and insight into what sparks people’s interest and get to connect with like minded folks from around the globe.

Promote a new service or product
You have a new product? It’s not there yet but you want to create some buzz? Or you’re holding it in your hands and want to let the whole world know how great it is and how you got there? Maybe you just wrote a book, or launched a website, added a new service, whatever it is – be it new and exciting or timeless – it gives you the right audience to share it with.

Reward your existing followers or clients with free advice on a topic you are knowledgeable in
You have thousands of loyal followers online, on social media or sitting in your database, clients, readers, the people, who make you who you are and what you are doing - worth it. What better way to thank them than give them a chance to ask you anything and share some free advice and guidance with them?

Test a new concept by allowing people to pick it to pieces with their questions
Have a great idea but you’re not sold on it? Host an AMA and see what people say! Two heads think better than one and hundreds surely even more so! By questioning things you can arrive to a much better solution quicker. Ask your existing followers and clients, ask our audience - we give you a great tool so use it to your advantage.

Connect with fellow professionals
Over 3000 experts in different fields from all over the world have shared their stories with us up to date. And this is just the beginning. You don’t get to talk to people like that every day (and for free!). Come ask, share, connect, learn and do better.

What's my Job?
I will be engaging with professionals from all walks of life in all the continent of the world about RChain. AMA gives us the platform to relate with interested individuals on the Crypto/blockchain channel with over 30000 participants to interact on issues concerning blockchain and RChain precisely.

Therefore I will be Hosting an RChain monthly event on the AMAfeed Crypto/Blockchain channel that will continually engage professionals all over the world including Co-hosting with several RAM label guides like @dckc @Jake-Gillberg @jimscarver @David405. @dckc for developers, will be answering questions and explaining some details of the RChain developers vision and concept to interested participants.

*Get in touch with a guide (https://github.com/rchain/bounties/wiki/Bounty-Task-Guides ) to align your contribution with goals of the RChain cooperative.*
*Replace these 3 lines with your answer*

### Budget and Objective

*First Event Host: *SCHEDULED on June 23, 2018 6:00 AM EDT:
 Event is live for 168hrs.
Here is the link to attend the Event: https://cryptoama.amafeed.com/user/52834
_Estimated Budget of Task:_ **$[400]**
_Estimated Timeline Required to Complete the Task:_ **[ COMPLETED]**

_How will we measure completion?_ **[event would have been held and audience answered and a link posted for confirmation of the event]**

*See CONTRIBUTING.md for details on budget and reward process.*

### Legal

_Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws._

**Comment by philipandri002** [comment]
**Update** of RChain Event ***

Event has started and we all ready have 171 likes and 15 comments.
You can still catch up with happen between now and tomorrow

**Comment by philipandri002** [comment]
Wow.. Event finally came to conclusion and it was awesome.

*Participant came from around the globe and we had s great time talking about the block chain/RCHAIN Technology.

*The event was live for 168hrs.
*I concluded by giving out the link posted by @pooreman #807 
* it was hectic but fun seeing people especially BlockChain enthusiasts show interest in the RChain Technology and ideology.

Here is the link to the event...

<a href="https://cryptoama.amafeed.com/ask-me-anything-about-rchain-blockchain-509429">RChainAMAEvent</a>

**Comment by Ojimadu** [comment]
@philipandri002 This issue has like 5 labels attached to it seems much IMO. 
I have reviewed the AMA; First, I think the name of the account should bear your name or something that would denote a community lead effort rather than an official one.
Most of the questions and answers were pretty much generic. I have doubts about the interest of the questionnaires IMO they seem to me to want to populate amafeed with a questions log rather than a genuine interest but whichever way, contents were generated that someone somewhere might find useful.
I voted $160 on this issue.

**Comment by philipandri002** [comment]
@Ojimadu."
I have doubts about the interest of the questionnaires IMO they seem to me to want to populate amafeed with a questions log rather than a genuine interest"  that sounds very funny to me, how you can make decision as serious as this on the basis of your assumption. You can't determine the credibility of such a wonderful event because of your mere suspicions.

I did a thorough research on the integrity of the AMA platform before hosting on the platform.

Secondly did you check on the crypto/ block chain channel and verify how many more blockchain platform have hosted also.. Maybe you should do more research

Also I can tell you q out the publicity of the AMAfeed platform about this event before and after the event.

We got 204 likes and even questions time didn't permit me to answer.

If will be very unfair for you to vote a $160 for such an effort. 
It was a seven days (168hrs)/event and time and data is valuable.

So I will ask you to please review your vote and be more reasonable.

Thanks

**Comment by philipandri002** [comment]
@ojimadu.. If you are sincere with your analysis you would agree that the label truly meets the aforementioned labels and that's why I did include them. I do what I know is right, if the issue fall into so many label category because of the nature I believe it is reasonable to let it carry the label.

Secondly you say "Most of the questions and answers were pretty much generic. I have doubts about the interest of the questionnaires IMO they seem to me to want to populate amafeed with a questions log rather than a genuine interest " I think sincerely speaking you're being naive or probably not a good researcher.. You would judge the effectiveness and intent of such an event on your baseless thought. With all due respect sir, when criticising peoples efforts you don't do it on the basis of assumptions, I will appreciate and I think everyone too will also want you to add some facts and a little evidence to back your microscopic opinion. But if it is on the basis of assumption I will kindly ask you to take back your criticism.

Having addressed your opinion let me state with fact, that the Host was a success. The Event was hosted on the crypto/blockchain channel. We had 204 likes over 52 interactions which was an awesome experience.

This event will always be a reference material in the crypto/blockchain channel in the AMAfeed platform for future reference to any interested reader or research as long as the earth exist.

I did a proper research and analysis about the AMAfeed platform and was convince it was a right place to host the RChain event.. A platform that grew from less than a thousand member last year to over thirty thousand is a formidable platform to associate with.

The platform as one of their incentives helps every host to do publicity for their event and I think RChain deserve such publicity

Last but no the least, check the content of the information that were published and exchanged. I did take out my time to ensure that the right information gets to my audience, including the video requested by pmoorman to be published see issue #807

I have more to say but let me drop my pen for now until due necessity but I believe I may have corrected your misinterpretation.

Thanks

**Comment by Ojimadu** [comment]
> @Ojimadu.. If you are sincere with your analysis you would agree that the label truly meets the aforementioned labels and that's why I did include them. I do what I know is right, if the issue fall into so many label category because of the nature I believe it is reasonable to let it carry the label.

On the labels, It was just my opinion nothing much, I just felt using lesser labels might be better.

> Secondly you say "Most of the questions and answers were pretty much generic. I have doubts about the interest of the questionnaires IMO they seem to me to want to populate amafeed with a questions log rather than a genuine interest " I think sincerely speaking you're being naive or probably not a good researcher.. You would judge the effectiveness and intent of such an event on your baseless thought. With all due respect sir, when criticising peoples efforts you don't do it on the basis of assumptions, I will appreciate and I think everyone too will also want you to add some facts and a little evidence to back your microscopic opinion. But if it is on the basis of assumption I will kindly ask you to take back your criticism...

You actually did put in some hours to get this done which is nice and should be rewarded and like I said earlier I believe someone somewhere might find it useful. However, 

- reviewing the profiles of a majority of those who asked the questions, asking questions seems to be what they do there. Like I said earlier, most of the questions and answers were pretty much generic. the questions especially seem to me to want to populate amafeed with a questions log rather than a genuine interest. Obviously, it is not your fault but the fact that there was no corresponding feedback on any of the answers from all the questions asked justifies my claim. Others can take a look too. 

- The "intro" and some answers you gave is a copy and paste from [coincentral](https://coincentral.com/rchain-beginner-guide) and some other sources. 

- the event had little engagement IMO to justify the budget. It has 56 comments and ~half of them came from you, and 6 questions (as at 02/07/2018 12:50 GMT+1) were left unanswered and you have already declared the event closed.

- You said the event was live for 168 hours meaning an average of 7.6 hours was spent to answer a question. Does this look right?

TL;DR: You have put in resources to get this done and deserve a reward, but the budget is not justifiable.

**Comment by philipandri002** [comment]
*reviewing the profiles of a majority of those who asked the questions, asking questions seems to be what they do there. Like I said earlier, most of the questions and answers were pretty much generic. the questions especially seem to me to want to populate amafeed with a questions log rather than a genuine interest. Obviously, it is not your fault but the fact that there was no corresponding feedback on any of the answers from all the questions asked justifies my claim. Others can take a look too.*
You say it seem like asking question is what they do there, I don't know how you plan to explain that with more fact it seem to me you enjoy your assumption approach to reality, however I was thinking you would say that the questions were asked by machines not real humans but since that is not the case here and that for whatever reason they asked they got the right answer. 

*The "intro" and some answers you gave is a copy and paste from coincentral and some other sources*
I would want to return this with a question sir.. Were the answers and intro wrong? Do they represent the facts and  true position of RChain? I would also ask you Sir; if there is anything wrong with coping an already existing fact that represent the true position of Rchain, wrong? I guess not. Don't forget it took me time to search, study and use those materials as answer only because I had studied it and find them relevant to answer such questions. Also you said they were copied from coincentral, I remember telling you never to make judgement on the basis of assumption but that seem to be a part of you and it will always produce for you false and inaccurate judgement. I would have been proven wrong by you if not that I actually copied from <a href="https://hashnode.com/post/rchain-cooperative-and-how-to-join-cjgwyi73l0010r3s1qtzi7a1e ">hashnode</a> . I still insist I don't see anything wrong if I discover an accurate article of RChain and use it for answer on related questions, if its against the rules of the RChain community please do endeavor to give me a pointer.

*the event had little engagement IMO to justify the budget. It has 56 comments and ~half of them came from you, and 6 questions (as at 02/07/2018 12:50 GMT+1) were left unanswered and you have already declared the event closed.*
It appears I have to keep reminding you that a proper research would have saved us slot of this stress. But let me educate you a little sir; The rules of AMA when hosting, is that you choose your starting time and closing time and once that its done it becomes automatic. What this means is that as soon as the time the event is slated to start, the events starts automatically and when the time you also programmed to end event ends automatically. Now the last few questions I couldn't answer came in very late and before I could answer them the event closed. So I didn't close the event, it happened automatically. However on this event it was opened for 168 hours.  Finally you sound like I just copied anything online and pasted. Do you know how many hours I had to put do study and analyse the documents to ensure it was worth sharing and relevant to the question asked.. I guess you see the work done was not a mere copy and paste.

@Ojimadu I sincerely appreciate your analysis and comment, and I believe it is necessary to help keep the coop healthy and I bear no personal grudge but my answers and response are basically necessary to prevent others from undervaluing my effort and commitment to the progress and  improvement of RChain cooperative.

Thanks

**Comment by philipandri002** [comment]
Budget Adjustment:

So if it were me i would put in a statement something like the following, in whatever words you yourself would choose [indicates fill in here]. 

"I opened this issue because i believe RChain Marketing should target a much wider audience. I still believe in that principle but in the present instance I didn't find great success in the AMA marketing. 

For this issue I'm changing my budget request to [$400] to help cover my time and expenses. I aim to continue to explore possibilities for strategic marketing and take risks where I feel there may be good but yet untried possibilities. Also I appreciate the feedback and discussions with @patrickM727 @pmoorman @AyAyRon-P

Thanks! 
Frank Nnalue

**Comment by pmoorman** [comment]
oke, there are basically 2 things that sit badly with me:

1. @philipandri002 is presenting as if he works for RChain, and thus represent us. This is _really_ harmful, and really shouldn't be encouraged like that. He's also making many factual errors such as saying that RChain transactions will be "99% secure" and that RHOC can be used only by RAMs, and stuff like that.  =>  red flag for me

2. All the people on the platform seem like robots, much as @Ojimadu points out above. Just open any 3 of them, and you'll see they engage on hundreds of AMAs on the platform asking all kind of basic questions about all sorts of things in life. I'm 99% sure that the vast majority these comments are essentially fake / gamed.

**Now, we get to the hard part:** 
In my book, the budget allocated should be proportional to the value created for RChain. Based on the 2 points above, I struggle to articulate what value exactly has been created, and it's easy to point out how potentially some of the answers (assuming they'd get traction) could get taken to be Truths™ published by an RChain representative and harm us.

**After discussion with @allancto I suggest the following:** 

- We allocate $150 for the effort that's been put in, which I belief has been in good faith and with the best of intentions (but maybe missing some crucial context)

- We (as a RAM community) need to set clearer standards as to whether to allocate budget based on "good intentions" and "resources invested", or just look at the value being created. Being a direct response marketer, I lean to the 2nd... but not everyone might agree. I suggest we open a discussion about effort-based or value-based rewards. (but first, I need to catch up with the guiding work, which is more urgent today).
